diff -Naur attrs-24.3.0/pyproject.toml attrs-24.3.0.patched/pyproject.toml
--- attrs-24.3.0/pyproject.toml	2024-12-16 19:26:35.000000000 +1300
+++ attrs-24.3.0.patched/pyproject.toml	2025-01-07 15:04:11.191607363 +1300
@@ -9,7 +9,7 @@
 name = "attrs"
 authors = [{ name = "Hynek Schlawack", email = "hs@ox.cx" }]
 license = "MIT"
-license-files = ["LICENSE"]
+license-files.globs = ["LICENSE"]
 requires-python = ">=3.8"
 description = "Classes Without Boilerplate"
 keywords = ["class", "attribute", "boilerplate"]
